packages/edit-post/src/hooks/components/media-upload/index.js

```diff
@@ -9,6 +9,22 @@ import { castArray, pick } from 'lodash';
 import { Component } from '@wordpress/element';
 import { __ } from '@wordpress/i18n';
 
+const getFeaturedImageMediaFrame = () => {
+	return wp.media.view.MediaFrame.Post.extend( {
```

Comment:
I think it would be better to extend the `Select` frame here instead of the `Post` frame so the sidebar isn't showing. The classic editor uses the `wp.media.featuredImage.frame()` function (which I didn't realize was a thing) to create the feature image flows for the media modal. [Here's where that happens](https://github.com/WordPress/wordpress-develop/blob/5af7160563b98a8687fa1948eb68abbccde1023c/src/js/_enqueues/wp/media/editor.js#L702-L714). The definition for the `wp.media.featureImage.frame()` [is here](https://github.com/WordPress/wordpress-develop/blob/5af7160563b98a8687fa1948eb68abbccde1023c/src/js/_enqueues/wp/media/editor.js#L640-L680) and it looks like that function includes some extra functionality for creating the toolbar/button correctly. I had a quick play at getting things hooked up this way, and haven't gotten it sorted yet. Hopefully this helps.

Reply:
Yeah, seeing the same. Theres a `createSelectToolbar` function that seems to not be getting triggered when we hit the frame. Still digging but thought I'd add that. Not sure how we'd make that happen yet...

https://github.com/WordPress/wordpress-develop/blob/6b366c6620fdd5960cedcdf80955966a715efa82/src/js/media/views/frame/post.js#L555-L560

Reply:
Also, setting defaults to match did not bring in the tollbar

```
				new wp.media.controller.FeaturedImage( {
					id: 'featured-image',
					title: wp.media.view.l10n.setFeaturedImageTitle,
					toolbar: 'featured-image',
					menu: 'default',
					multiple: false,
					filterable: 'uploaded',
					priority: 60,
					syncSelection: true,
				} ),
```

Reply:
I'm still not understanding why select is not loading the button...I think for the sake of getting this in before the release, we should move forward with the Post flow and open an enhancement issue on the sidebar. 

Would you agree @joemcgill ? Open to any suggestions.